components/scripts/lib/git.sh

```diff
@@ -29,8 +29,8 @@ git_clone_project() {
    debug git clone --depth=1 ${git_options} ${branch} "${git_repo}" "${clone_dir}"
 
    # shellcheck disable=SC2086  # we want $git_options and $branch to expand into multiple arguments
-   git clone --depth=1 ${git_options} ${branch} "${git_repo}" "${clone_dir}" || die "ERROR: Unable to clone from ${git_repo}." 2
-   cd "${clone_dir}" || die "Unable to access git repository directory ${clone_dir}." 2
+   git clone --depth=1 ${git_options} ${branch} "${git_repo}" "${clone_dir}" || die "ERROR: Unable to clone from ${git_repo}."
```

Comment:
I decided it's better to not provide an explicit exit code here and instead rely on the default (which is `UNEXPECTED_ERROR`) because that pattern is easier to follow if we add additional `die` calls in the future.